src/components/button/common/button-common.scss

```diff
@@ -185,25 +96,9 @@ $icon-only: ':where([data-slot-names~=icon], [icon-name]):not([data-slot-names~=
   }
 }
 
-:host([data-icon-small]) {
-  @include sbb.icon-button-variables;
-
-  --sbb-button-shadow-1-offset-y: var(--sbb-shadow-elevation-level-3-shadow-1-spread);
-  --sbb-button-shadow-2-offset-y: var(--sbb-shadow-elevation-level-3-shadow-1-spread);
-  --sbb-button-shadow-1-blur: var(--sbb-shadow-elevation-level-3-shadow-1-spread);
-  --sbb-button-shadow-2-blur: var(--sbb-shadow-elevation-level-3-shadow-1-spread);
-  --sbb-button-shadow-1-color: var(--sbb-color-black-alpha-0);
-  --sbb-button-shadow-2-color: var(--sbb-color-black-alpha-0);
-}
-
-:host([data-icon-small][negative]) {
-  @include sbb.icon-button-variables-negative;
-
-  --sbb-button-shadow-1-color: var(--sbb-color-black-alpha-0);
-  --sbb-button-shadow-2-color: var(--sbb-color-black-alpha-0);
-}
-
-:is(.sbb-button, .sbb-button-link, .sbb-button-static) {
+[class$='-link'],
+[class$='-static'],
+[class$='-button'] {
```

Comment:
This maybe needs some checks after the #2431 has been merged (use new class 'sbb-action-base`?)

Reply:
Yes, I would argue to use `sbb-action-base` to avoid selector lists.
What do you think?

Reply:
Sure, I'll wait the merge to rebase and fix it 👍